src/visualization/vega_renderer/JSCanvas.m

```diff
@@ -627,7 +626,11 @@ - (void)fill {
     NSArray *args = [JSContext currentArguments];
     (void)args;
     assert(args != nil);
-    assert(args.count == 0);
+    if(args.count > 0) {
```

Comment:
I think we can just remove this entirely at some point (as discussed offline, doesn't need to be addressed in this PR).